src/app/BE/card/templates/Card.js

```diff
@@ -35,7 +35,27 @@ const DefaultCard = ({ parentCode, actions = [], code, columns }) => {
   const userCode = useSelector(selectCode('USER'), equals)
   const userType = getUserType(useSelector(selectCode(userCode), sameLength))
 
-  return (
```

Comment:
This is a generic card used across all projects of alyson, it shouldn't have project specefic name.